cmd/datasource-syncer/README.md

```diff
@@ -0,0 +1,63 @@
+# Datasource Syncer
+
+This CLI tool can act as a cron job remotely syncing the given Grafana Prometheus datasource, so it has e.g.:
+
+* Correct URL
+* Correct HTTP method
+* Fresh Oauth2 access token (valid for 1h)
+
+This allows you to directly query Google Cloud Monitoring with Grafana.
```

Comment:
By regularly refreshing the oAuth2 access token, you can configure Grafana to directly query Google Cloud Monitoring (Managed Service for Prometheus).